listen.gemspec

```diff
@@ -33,6 +33,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'rb-fsevent', '>= 0.9.3'
   s.add_dependency 'rb-inotify', '>= 0.9.7'
 
+  # Used to show warnings at runtime
+  s.add_dependency 'ruby_dep', '~> 1.1'
```

Comment:
@e2 What is the reason you have to check the ruby version when you do `bundle install` in dev mode? Would it help to add a `ruby_version` to the gem file? In that case you can let Bundler take care of it.